### PR TITLE
[BUG, MRG] Fix pial surface loading, logging in locate_ieeg

### PR DESCRIPTION
**Pull request by alexrockhill**
Fixes https://github.com/mne-tools/mne-python/issues/11484

### CODE REVIEW

**Comment by alexrockhill** [comment]
Here's a minimal script to check

```
import os.path as op
import numpy as np

import nibabel as nib
from dipy.align import resample

import mne

misc_path = mne.datasets.misc.data_path()
subject = 'sample_seeg'
subjects_dir = op.join(misc_path, 'seeg')

CT_orig = nib.load(op.join(misc_path, 'seeg', 'sample_seeg_CT.mgz'))
raw = mne.io.read_raw(op.join(misc_path, 'seeg', 'sample_seeg_ieeg.fif'))

T1 = nib.load(op.join(misc_path, 'seeg', 'sample_seeg', 'mri', 'T1.mgz'))
reg_affine = np.array([
    [0.99270756, -0.03243313, 0.11610254, -133.094156],
    [0.04374389, 0.99439665, -0.09623816, -97.58320673],
    [-0.11233068, 0.10061512, 0.98856381, -84.45551601],
    [0., 0., 0., 1.]])
CT_aligned = mne.transforms.apply_volume_registration(CT_orig, T1, reg_affine)

trans = mne.coreg.estimate_head_mri_t(subject, subjects_dir)
gui = mne.gui.locate_ieeg(raw.info, trans, CT_aligned, subject=subject, subjects_dir=subjects_dir)
```

**Comment by alexrockhill** [comment]
Ok to merge @drammock @mmagnuski?

File: mne/gui/_core.py [hunk 1]
```diff
@@ -165,12 +165,15 @@ def _load_image_data(self, base_image=None):
                  'head plot, use :ref:`mne.bem.make_scalp_surfaces` '
                  'to add the scalp surface instead of skull from the CT')
             self._head = None
-        if op.exists(op.join(self._subject_dir, 'surf', 'lh.pial')):
-            self._lh = _read_mri_surface(
-                op.join(self._subject_dir, 'surf', 'lh.pial'))
+        for img in ('', '.T1', '.T2', ''):
```
Comment (drammock) [inline mne/gui/_core.py:168]: you've used this trick elsewhere too, right? The extra `''` at the end just makes the error message cleaner? If that's right, can you just add a comment explaining, commit with `ci skip`, then merge?

**Review by drammock — APPROVED** [review]

File: mne/gui/_core.py (hunk 1 above)
Comment (alexrockhill) [reply mne/gui/_core.py:168]: Yes, that's correct, will do

**Pull request merged**